NEWS.md

```diff
@@ -584,3 +585,4 @@ Deprecated or removed
 [#20543]: https://github.com/JuliaLang/julia/issues/20543
 [#20609]: https://github.com/JuliaLang/julia/issues/20609
 [#20889]: https://github.com/JuliaLang/julia/issues/20889
+[#21359]: https://github.com/JuliaLang/julia/issues/21359
```

Comment:
Good catch! I stripped this line from the diff here and elsewhere. I will strip this line from any future commits as well. Thanks!